client/blocks/reader-full-post/style.scss

```diff
@@ -412,6 +412,24 @@
 	margin: 28px 0 26px;
 }
 
+.reader-full-post .embed-youtube,
+.reader-full-post .embed-vimeo {
+	display: block;
+	position: relative;
+	padding: 25px 0 56.25%;
+
+	iframe {
+		height: 100%;
+		position: absolute;
+			top: 0;
```

Comment:
@samouri We indent `top/right/bottom/left` one level below the position declaration.